packages/react-linked-input/LinkedInput.js

```diff
@@ -10,7 +10,7 @@
 'use strict';
 
 var React = require('react');
-var LinkedValueUtils = require('react/lib/LinkedValueUtils');
+var LinkedValueUtils = require('react-dom/lib/LinkedValueUtils');
```

Comment:
Probably don't need to update this package - I think this was a once and done release. (right @jimfb?)